src/prng.rs

```diff
@@ -14,7 +14,7 @@ use getrandom::getrandom;
 
 use std::marker::PhantomData;
 
-const BUFFER_SIZE_IN_ELEMENTS: usize = 128;
+const BUFFER_SIZE_IN_ELEMENTS: usize = 32;
```

Comment:
Much lower than this and the `prng` benchmarks begin to regress for longer outputs.